pkg/apis/pipeline/v1beta1/pipelinerun_types.go

```diff
@@ -561,6 +561,7 @@ func (pr *PipelineRun) GetTaskRunSpec(pipelineTaskName string) PipelineTaskRunSp
 			s.StepOverrides = task.StepOverrides
 			s.SidecarOverrides = task.SidecarOverrides
 			s.Metadata = task.Metadata
+			s.ComputeResources = task.ComputeResources
```

Comment:
could you add a test to make sure that the taskrun is created with the compute resources for the pipelinerun's taskrunspecs?

Reply:
yap

Reply:
moved this change and added the test case in the following PR 
https://github.com/tektoncd/pipeline/pull/5082

Reply:
I think this test case is in scope for this PR actually

Reply:
added the test case back in current PR.